docs/changes.rst

```diff
@@ -118,7 +118,7 @@ Released: not yet
 * Made the `MOFWBEMConnection` class internal and undocumented. It has an
   inconsistent semantics and should not be used by users. (See issue #2001).
 
-* Removed the `FakedWBEMConnection` init argument `conn_lite`. The lighe mode
+* Removed the `FakedWBEMConnection` init argument `conn_lite`. The lite mode
```

Comment:
This fix has been merged as part of the review of the incompatible changes. Please rebase this PR to see whether it creates a merge conflict.

Reply:
TODO: